Sources/Widgets/Core/DragHelper/index.d.ts

```diff
@@ -0,0 +1,61 @@
+import { vtkObject } from '../../../interfaces';
+import { Nullable, Vector3 } from '../../../types';
+
+export interface IWidgetDragHelperInitialValues {
+  lastOrigin?: Vector3;
+}
+
+export interface vtkWidgetDragHelper extends vtkObject {
+  /**
+   * Sets the previous drag interaction origin.
+   * @param {Nullable<number[]>} origin the previous drag event's world coordinates, if any
+   */
+  setLastOrigin(origin: Nullable<number[]>): boolean;
+
+  /**
+   * Gets the previous drag interaction origin.
+   */
+  getLastOrigin(): Nullable<Vector3>;
+
+  /**
+   * Calculates the delta from the last origin to the given coordinate.
+   * @param {Nullable<number[]>} dragWorldCoords the next drag interaction coordinate, if any
+   */
+  calcDelta(dragWorldCoords: Nullable<number[]>): Vector3;
+
+  /**
+   * Calculates the new position from a starting position given a drag interaction coordinate.
```

Comment:
Can you give more explanations in this documentation ? It's not really clear what it does.